### PR TITLE
Update the documentation about the devApplySettings function

### PR DESCRIPTION
**Pull request by Senyoret1**
Changes:
- The documentation about the `devApplySettings` function was updated.

Does this change need to mentioned in CHANGELOG.md?
No

Requires testing
No

Comments about testing , should you have some
Nothing

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -139,22 +139,25 @@ devApplySettings(usePassphrase, deviceLabel, pinCodeReader)
 
 *Purpose:*
 
-Enable or disable the passphrase protection.
+Change parts of the device configuration.
```
Comment (olemis) [inline DOCUMENTATION.md]: What about `Edit device configuration` ?

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -139,22 +139,25 @@ devApplySettings(usePassphrase, deviceLabel, pinCodeReader)
...
 
 *Params:*
-- usePassphrase: Indicates if the passphrase protection must be enabled (true) or disable (false).
-- deviceLabel: Label to identify the device through application and/or device screen.
+- usePassphrase: Indicates if the passphrase protection must be enabled (true) or disable (false). Use `null` to
```
Comment (olemis) [inline DOCUMENTATION.md]: Whata bout `If set to `null` this value will be ignored and passphrase will remain unchanged` ?

File: DOCUMENTATION.md [hunk 3]
```diff
@@ -139,22 +139,25 @@ devApplySettings(usePassphrase, deviceLabel, pinCodeReader)
...
+not make changes to this configuration.
```
Comment (olemis) [inline DOCUMENTATION.md]: single line

File: DOCUMENTATION.md [hunk 4]
```diff
@@ -139,22 +139,25 @@ devApplySettings(usePassphrase, deviceLabel, pinCodeReader)
...
+- deviceLabel: Label to identify the device through application and/or device screen (32 characters or less). Use
```
Comment (olemis) [inline DOCUMENTATION.md]: Suggestions similar to `usePassphrase`.

**Comment by Senyoret1** [comment]
Integrated in https://github.com/skycoin/hardware-wallet-js/pull/83

**Pull request closed without merging**